Sources/Afluent/AUOWCacheStrategies/AUOWCache+Strategy.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  AUOWCache+Strategy.swift
+//
+//
+//  Created by Annalise Mariottini on 12/17/24.
+//
+
+import Foundation
+
+extension AUOWCache {
+    /// ``Strategy`` represents the available caching strategies for the ``AUOWCache``.
+    public protocol Strategy {
```

Comment:
TIL https://github.com/swiftlang/swift-evolution/blob/main/proposals/0404-nested-protocols.md

(pleasantly surprised)

Reply:
Ahh this is amazing!!! And unfortunately Swift 5.10+, so we have to decide whether we'll drop 5.9 support.

Reply:
Hrm, which do you prefer:
* Higher likelihood of consumer build issues if they were using `AUOWCache.Strategy` to define the cache strategy locally (unlikely, since until literally a few days ago there was only 1 and the pattern was always to define it directly as the param, but still a possibility)
* Keeping 5.9 support

I guess since we're still in v0 land the former isn't too big of a deal

Reply:
Keeping 5.9 support. Because to your point, we're still v0

Reply:
Ok cool, I'll make that change later today or tomorrow

Reply:
I had to rebase off of main anyway due to the recent revert. Change made to no longer nest the protocol.